app/controllers/api/v1/scholarship_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class ScholarshipApplicationsController < ApplicationController
+      respond_to :json
+
+      def create
+        @scholarship_application = ScholarshipApplication.new(reason: params[:reason], terms_accpeted: params[:terms_accepted], scholarship_id: params[:scholarship_id], user_id: current_user.id)
+        if @scholarship_application.save
+          render json: { redirect_to: '/success' }
+        else
+          render json: { errors: @scholarship_application.errors.values }, status: :bad_request
```

Comment:
Since this `@scholarship_application` instance variable is only being used locally, and does not need to be accessible in a view, it can just be a local variable, i.e. `scholarship_application`.